quickstart/bin/re-render.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+GALOY_DEV_DIR=${REPO_ROOT}/quickstart/dev
+GALOY_ROOT_DIR=${REPO_ROOT}/quickstart/galoy
```

Comment:
this is not very clear to me. why root dir is a sub directory of quickstart?

Reply:
This is just for within the context of `re-render`. It syncs the state of the main repo into `quickstart/galoy`. It's like a snapshot of the latest stable / working / released state of the galoy repo as it was in a situation that we know Quickstart is working.

The idea here is that
```
git clone <galoy-repo>
cd quickstart
docker compose up -d
```

Should remain stable even if there are untested / unreleased commits on the main branch.
`bin/re-render.sh` will be executed as part of the release process once tests have past.
We don't want to reference the files under the 'real' repo root as they have not been vetted yet.

Reply:
Its the same technique we use in Galoy-infra where the examples should always point to the latest release of the repo and not the head.

See https://github.com/GaloyMoney/galoy-infra/blob/0fcd4c139a4b1051e7f85ecd25f2b56dcd9721ca/examples/gcp/platform/main.tf#L15